### DISCUSSION TITLE
Control phone terminal

### DISCUSSION THREAD

**Discussion by Jimocu** [Malcolm-Q/LC-LateGameUpgrades#452] [Ideas]
Hello, I was thinking of a single-player game (which could even be used by several players), an item like a telephone on which you could enter codes like on the terminal (for opening/closing doors, in this case). 
It wouldn't allow you to deactivate turrets and mines because the codes aren't written on them.

**Comment by WhiteSpike** [comment]
So like a terminal but it would only allow you to input codes?
The logic I can think of for this is probably like some item like a screen maybe which you could inspect and that would activate the input for that screen for codes. Once typed and entered, it would execute the same logic as the terminal sending a code.
If this is feasible or not, I am not sure but the logic for it doesn't seem hard to do.

**Comment by Jimocu** [comment]
And there could be different levels of improvement, for example level 1, only allowing you to interact with doors, level 2 to see and deactivate turrets and mines in the user's vicinity, and a level 3 that would allow you to control objects from this screen to save time during the day.

As my friends don't want to try hard on this game, I like to play single-player games with mods that allow progressive progression (LGU and Lethal progression add up well, I think).

Translated with DeepL.com (free version)
